primer/brand
# Minimal Footer Updates

## Summary

There were some layout issues adding polymorphism to the MinimalFooter.Link component. To address this I have updated the CSS to reflect the changes and updated the Storybook docs so we will have visual regression tests.

## List of notable changes:

- **updated** MinimalFooter.Link CSS to specify padding on all sides to avoid browser-implied styles for buttons.

## What should reviewers focus on?

- Ensuring no additional padding is added for the MinimalFooter.Link component when using it as a Button

## Steps to test:

1. Open the MinimalFooter component in CI-deployed preview environment
2. Go to Mixed Buttons and LInks story in Storybook
3. Verify that each link item behaves as expected.

## Supporting resources (related issues, external links, etc):

- [Slack Thread](https://github.slack.com/archives/C03FY9G00NS/p1684171676600209)

## Contributor checklist:


## Reviewer checklist:
